Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -2781,10 +2790,22 @@ private void LynchCycle()
                         var t = choices.FirstOrDefault(x => x.PlayerRole == IRole.Tanner);
                         if (t != null && t.Votes > 0)
                             AddAchievement(t, AchievementsReworked.SoClose);
+                        var ScapeGoat = Players.FirstOrDefault(x => x.PlayerRole == IRole.ScapeGoat && !x.IsDead);
+                        if (ScapeGoat != null)
+                        {
+                            SendWithQueue(GetLocaleString("ScapeGoatLynched", lynched.GetName()));
+                            KillPlayer(lynched, KillMthd.Lynch, killers: Players.Where(x => x.Choice == lynched.Id), isNight: false);
+                        }
                     }
                     else // if (lynched.Votes == -2) // No lynch votes at all
                     {
                         SendWithQueue(GetLocaleString("NoLynchVotes"));
+                        var ScapeGoat = Players.FirstOrDefault(x => x.PlayerRole == IRole.ScapeGoat && !x.IsDead);
+                        if (ScapeGoat != null)
```

Comment:
Local variable names should be `snakeCase` in C#
```suggestion
                        var scapeGoat = Players.FirstOrDefault(x => x.PlayerRole == IRole.ScapeGoat && !x.IsDead);
                        if (scapeGoat != null)
```